apps/src/lab2/lab2Redux.ts

```diff
@@ -14,9 +14,9 @@ import {
   AppName,
   Channel,
   LevelData,
-  LevelProperties,
   ProjectManagerStorageType,
   ProjectSources,
+  LevelProperties,
```

Comment:
Nit: why did you move this (out of alpha order)?

Reply:
Oh good catch, no particular reason :) I think I must have removed the import then re-added it